certbot/tests/error_handler_test.py

```diff
@@ -112,6 +113,59 @@ def test_sysexit_ignored(self):
             pass
         self.assertFalse(self.init_func.called)
 
+    @mock.patch('certbot.main.sys')
+    def test_handle_exception(self, mock_sys):
```

Comment:
I looks like the purpose of `error_handler.ErrorHandler` is to handle clean up of stateful operations when there is an exception. But `main._handle_exception` is for logging tracebacks and displaying them to the user properly. So they are separate but sound similar.

I think it would be better to move this test into its own class in main_test.py.